feg/gateway/diameter/diameter_client.go

```diff
@@ -278,9 +278,11 @@ func (client *Client) DisableConnectionCreation(period time.Duration) {
 // corresponding answer. Additionally, SendRequest will add the OriginHost/Realm
 // AVPs to the message because they are mandatory for all requests
 // Input: server - cfg containing info on what server to send to
-// 				done - channel to send the answer to when received
-//				message - request to send
-//				key - something to uniquely identify the request
+//
+//	done - channel to send the answer to when received
+//	message - request to send
+//	key - something to uniquely identify the request
```

Comment:
There seem to be quite a few cases where lists of parameters are formatted in a way the Go formatter doesn't like, but it seems the 1.19 formatter is making the situation worse. Should this be fixed in this PR by e.g. making these into lists so the formatter doesn't complain or leaving it to a follow up?

Reply:
I would prefer to fix it in a follow up PR.

Reply:
I actually don't see it in this example (and haven't noticed any really bad formattings going through the review). The new formatting seems okay to me.

Reply:
Here is the compete example: 
https://github.com/magma/magma/blob/a9a62c98a7fb2bfe131c8c8dd06345153553652f/feg/gateway/diameter/diameter_client.go#L280-L286

Reply:
Yes, exactly. Like this the four input parameters are separated into two groups

Reply:
Ah, right, I missed the newline and only noticed the change to the indentation. Yes, that's clearly worse than before.

Reply:
But there are also deleted indentations like here:
![image](https://user-images.githubusercontent.com/6066106/200879120-517eb025-90f9-4c9a-88fe-9dec7a21abbe.png)

Reply:
Since you already found examples, maybe you can already start a follow-up (or even push the fix here, I don't have a clear preference).